memcached_prefix

```diff
@@ -0,0 +1 @@
+689f5ba0a0bab644aaa29626c326d455:
```

Comment:
this shouldn't be being committed (it's in .gitignore; I'm a bit surprised this was possible at all).